include/caffe/util/math_functions.hpp

```diff
@@ -44,20 +44,20 @@ void caffe_cpu_eltwise_min(const int N, const Dtype alpha, const Dtype* X,
     const Dtype beta, Dtype* Y);
 
 template <typename Dtype>
-void caffe_copy(const int N, const Dtype *X, Dtype *Y);
+void caffe_copy(const int N, const Dtype* X, Dtype* Y);
```

Comment:
We didn't write this:
https://github.com/NVIDIA/caffe/blame/v0.15.1/include/caffe/util/math_functions.hpp#L47